### PR TITLE
Clarify process instantiation REST API

### PR DESCRIPTION
**Pull request by goenkale**
fix(api) Clarify the differences between CASE resource and PROCESS (resource regarding process instantiation using contract)

### CODE REVIEW

**Comment by educhastenier** [comment]
retest this please

**Comment by educhastenier** [comment]
retest this please

File: md/bpm-api.md [hunk 1]
```diff
@@ -3291,7 +3291,7 @@ Note: if the `userId` is not provided as a deploy parameter, the `userId` proper
 
 #### Description
  
-Deploy and manage [process definitions](key-concepts.md) In addition, you can instantiate a process, which will create a new process instance ([case](#case)).
+Deploy and manage [process definitions](key-concepts.md) In addition, you can instantiate a process, which will create a new process instance ([case](bmp-api.md#case)).
```
Comment (educhastenier) [inline md/bpm-api.md]: This is not the problem with this link: ANCHOR "case" does not exist, so the link won't work.

File: md/bpm-api.md [hunk 2]
```diff
@@ -3531,9 +3536,10 @@ Process instantiation contract elements can be retrived client side.
       ]
     }
     ```
-<a id="start-process-with-contract"/>
 
-#### Start a process using an instantiation contract
+##### Start a process using an instantiation contract
+
+This method will create a new process instance ([case](bpm-api.md#case))
```
Comment (educhastenier) [inline md/bpm-api.md]: Dead link

File: md/bpm-api.md [hunk 3]
```diff
@@ -2814,13 +2810,18 @@ You can get a case by using its identifier. Request url
...
   ```json
   {
     "processDefinitionId":"5777042023671752656"
   }
   ```
-  Create a case with variables
+##### Create a case with variables
+
+::: danger
+:fa-exclamation-triangle: **Warning:** The attribute "variables" on the request payload is used to initialize the process variables (not BDM variables).
```
Comment (educhastenier) [inline md/bpm-api.md:2825]: I would add `If you want to initialize BDM variables at process instantiation, add a contract on the process and map BDM variables to the contract data. See [Start a process using an instantiation contract](#start-process-with-contract) for usage.`

**Comment by educhastenier** [comment]
retest this please

File: md/bpm-api.md (hunk 2 above)
Comment (educhastenier) [reply md/bpm-api.md]: Try removing the `bpm-api.md` because it still does not work

File: md/bpm-api.md [hunk 4]
```diff
@@ -3289,8 +3293,8 @@ Note: if the `userId` is not provided as a deploy parameter, the `userId` proper
 ### Process
 
 #### Description
-
-Manage a [process definition](key-concepts.md) (not to be confused with a [case](#case), which is a process instance).
+ 
+Deploy and manage [process definitions](key-concepts.md) In addition, you can instantiate a process, which will create a new process instance ([case](bmp-api.md#case)).
```
Comment (educhastenier) [inline md/bpm-api.md]: Try removing the `bpm-api.md` because it still does not work

**Comment by educhastenier** [comment]
retest this please

**Comment by abirembaut** [comment]
retest this please

File: md/bpm-api.md [hunk 5]
```diff
@@ -3230,7 +3234,7 @@ Note: if the `userId` is not provided as a deploy parameter, the `userId` proper
 #### Search for comments
 
 * **URL**  
-  `/API/bpm/comment`  
+  `/API/bpm/case`
```
Comment (educhastenier) [inline md/bpm-api.md:3237]: please change back to `/API/bpm/comment`

File: md/bpm-api.md [hunk 6]
```diff
@@ -3289,8 +3293,8 @@ Note: if the `userId` is not provided as a deploy parameter, the `userId` proper
 ### Process
 
 #### Description
-
-Manage a [process definition](key-concepts.md) (not to be confused with a [case](#case), which is a process instance).
+ 
+Deploy and manage [process definitions](key-concepts.md) In addition, you can instantiate a process, which will create a new process instance ([case](#case)).
```
Comment (educhastenier) [inline md/bpm-api.md:3297]: you're missing a `.` before `In addition`

**Comment by abirembaut** [comment]
retest this please

**Comment by educhastenier** [comment]
Merged. See commit bc67050b7d4d191c1fe6874b410fad5d0a14f461.

**Pull request closed without merging**